test/e2e/checkpoint_test.go

```diff
@@ -63,6 +60,13 @@ var _ = Describe("Podman checkpoint", func() {
 		result := podmanTest.Podman([]string{"container", "checkpoint", cid})
 		result.WaitWithDefaultTimeout()
 
+		cmd := exec.Command("cat", podmanTest.CrioRoot+"/vfs-containers/"+cid+"/userdata/dump.log")
+		out, _ := cmd.CombinedOutput()
+		fmt.Printf("combined out:\n%s\n", string(out))
+		cmd = exec.Command("ausearch", "-m", "AVC", "-m", "USER_AVC", "-ts", "recent")
+		out, _ = cmd.CombinedOutput()
+		fmt.Printf("combined out:\n%s\n", string(out))
+
```

Comment:
Please feel free to do whatever debugging you need to in the CI scripts as well.  For example, ``set +e`` before the test runs/fails, then ``journalctl -lb 0`` to splat the logs.